misaligned.js

```diff
@@ -11,6 +11,12 @@ function print_color_map() {
     return majorColors.length * minorColors.length;
 }
 
+function computePairNumber(majorIndex,minorIndex){
+    return majorIndex * 5 + minorIndex
+}
+
 result = print_color_map();
 expect(result).equals(25);
+expect(computePairNumber(0,0)).equals(1)

```

Comment:
This will test the functionality of `computePairNumber`. But this function is not used in the printing function.
That means when we fix this function, the test will pass, but the customer will still get the wrong report.
Can you change the `print_color_map` function, to use `computePairNumber`?